trawler.py

```diff
@@ -31,12 +37,14 @@ class Trawler(object):
     # Default to True, but detected unless overridden in config
     use_kubeconfig = True
     # Default path for secrets in container build - override with envvar SECRETS
+    mtls = False
     secrets_path = '/app/secrets'
     graphite = None
     gauges = {}
 
-    def __init__(self, config_file=None):
+    def __init__(self, config_file=None, ):
         self.secrets_path = os.getenv('SECRETS', self.secrets_path)
+        self.mtls = os.getenv("HAS_MTLS_ENABLED", 'False').lower() in ('true', '1', 't')
```

Comment:
I think ENABLE_MTLS is better as we're telling Trawler what to do with this environment variable
```suggestion
        self.mtls = os.getenv("ENABLE_MTLS", 'False').lower() in ('true', '1', 't')
```